web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -54,7 +55,7 @@
   });
 
   const handleKeyboardPress = (event: KeyboardEvent) => {
-    if ($isSearchEnabled) {
+    if ($isSearchEnabled || $disableShortcut) {
```

Comment:
I don't think this is a great solution personally. Could you not just check if the target element is a text or input field?

Reply:
Yes it would be better. How can I revert this change since it has been merged ?

Reply:
You could easily use this same branch though.